src/main/resources/assets/js/application/recruiter/views/tdpr-directive-slot.html

```diff
@@ -1,3 +1,3 @@
 <td class="slot" ng-class="getClass()">
-    <span class="slot-span" uib-tooltip="{{slotData.type}}"></span>
-</td>
\ No newline at end of file
+    <span class="slot-span" uib-tooltip="{{getTooltipString()}} {{slotData.type}}"></span>
```

Comment:
Could we put slotData.type to getTooltipString() function? Would be easier to manipulate it later.